frame/nft/src/test/mock.rs

```diff
@@ -0,0 +1,81 @@
+#![cfg(test)]
+
+use composable_tests_helpers::test::block::{process_and_progress_blocks, MILLISECS_PER_BLOCK};
+use frame_support::{
+	parameter_types,
+	traits::{ConstU64, Everything},
+};
+use frame_system as system;
+use sp_core::H256;
+use sp_runtime::{
+	testing::Header,
+	traits::{BlakeTwo256, IdentityLookup},
+};
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+// Configure a mock runtime to test the pallet.
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		Timestamp: pallet_timestamp::{Pallet, Call, Storage},
+		Nft: crate::{Pallet, Storage , Event<T>},
+	}
+);
+
+impl crate::Config for Test {
```

Comment:
i prefer Runtime word instead of Test. code is more polymorphic than with benchmarks and real runtimes.